tests/Unit/Emogrifier/HtmlProcessor/CssInlinerTest.php

```diff
@@ -4,6 +4,7 @@
 
 use Pelago\Emogrifier;
 use Pelago\Emogrifier\CssInliner;
+use Pelago\Tests\Unit\Emogrifier\Support\AssertCss;
```

Comment:
Please let's move the trait to the namespace `Pelago\Tests\Support\Traits` (so a relatively high level, and with `Traits` added).

Reply:
Ok.  Should the corresponding test then be in `Pelago\Tests\Unit\Tests\Support\Traits`?

Reply:
Um … :-)

The tests for the trait should be in `Pelago\Tests\Unit\Support\Traits` (so only one ` Tests` ). (Tests for stuff in tests seems to resists namespacing that is both sane and consistent.)